homeassistant/components/iaqualink/__init__.py

```diff
@@ -101,3 +117,36 @@
     hass.data[DOMAIN].clear()
 
     return all(await asyncio.gather(*tasks))
+
+
+def refresh_system(func):
+    """Force update all entities after state change."""
+
+    @wraps(func)
+    async def wrapper(self, *args, **kwargs):
+        """Call decorated function and send update signal to all entities."""
+        await func(self, *args, **kwargs)
+        async_dispatcher_send(self.hass, DOMAIN)
+
+    return wrapper
+
+
+class AqualinkEntity(Entity):
+    """Abstract class for all Aqualink platforms."""
+
+    async def async_added_to_hass(self) -> Awaitable[None]:
```

Comment:
Copied from homeassistant/components/geniushub/__init__.py. I've always been under the impression that not returning ended up returning None. Looks like typing is pretty inconsistent across components, some declare the return value as 'None' instead of 'Awaitable[None]'.

I'll make the requested change anyway, not caring either way.

Reply:
Ok, just realized you didn't mean I should "return None" but rather change "Awaitable[None]" to "None"...